util/src/main/java/io/kubernetes/client/util/ClientBuilder.java

```diff
@@ -117,6 +121,24 @@ private static File findConfigFromEnv() {
     }
   }
 
+  private static class KubeConfigEnvParser {
+    private String parseKubeConfigPath(String kubeConfigEnv) {
+      if (kubeConfigEnv == null) {
+        return null;
+      }
+
+      final String[] filePaths = kubeConfigEnv.split(File.pathSeparator);
+      final String kubeConfigPath = filePaths[0];
+      if (filePaths.length > 1) {
+        log.debug(
+            "Found multiple kubeconfigs files, $KUBECONFIG: " + kubeConfigEnv + " using first: {}",
```

Comment:
Hrm, isn't debug even lower level than warning? I think this should be warning.

Reply:
Sorry, I have not understood. Fixed.